app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getTopics } = require("./Controllers/api.controller");
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((request, response, next) => {
+    response.status(404).send({ msg: "Path not found" });
```

Comment:
Nice work! Just for future reference, express does have an inbuilt behaviour to handle route 404s by default, so you could just expect a 404 status code to cover this task instead of writing middleware for it (if you're curious try commenting this code out - you should still get a 404 status code!)